packages/n/noto-sans-ttf/package.yml

```diff
@@ -41,5 +41,9 @@ install    : |
 
     install -dm00755 $installdir/usr/share/fonts/conf.d
     ln -sv /usr/share/fontconfig/conf.avail/66-noto-color-emoji.conf $installdir/usr/share/fonts/conf.d/66-noto-color-emoji.conf
+
+    install -Dm00644 $pkgfiles/noto-sans-ttf.metainfo.xml $installdir/usr/share/metainfo/noto-sans-ttf.metainfo.xml
+
+    install -Dm00644 $pkgfiles/noto-serif-ttf.metainfo.xml $installdir/usr/share/metainfo/noto-serif-ttf.metainfo.xml
```

Comment:
Needs to be patterned into the noto-serif-ttf

Reply:
Added. Verified with testing that it works.